app/controllers/PathManagerController.scala

```diff
@@ -81,4 +94,6 @@ object PathManagerController extends Controller {
     Ok(s"current id = $currentId")
   }
 
+  def argoOk(json: JsValue) = Ok(Json.obj("data" -> json)).as("application/vnd.argo+json")
```

Comment:
It's our media-type! :-)

Would be good to link to https://github.com/argo-rest/spec from the README.